src/Symfony/Component/Process/Tests/KillableProcessWithOutput.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$outputs = array(
+    'First iteration output',
+    'Second iteration output',
+    'One more iteration output',
+    'This took more time',
+    'This one was sooooo slow',
+);
+
+$iterationTime = 10000;
+
+foreach ($outputs as $output) {
+    usleep($iterationTime);
+    $iterationTime *= 10;
```

Comment:
Won't this create an extremely slow test?

Reply:
The test should end after 1s and 110ms of execution of this code. If a failure occurs, the timeout will be reached 60s after the start of the test.

The process is long to end (but ends, for "security" purpose) to simulate the running of a daemon.

Note: for some reason under windows it looks like this test takes 4s. No idea why and how. Any though about?

Windows PHPUnit output on test process:
```
1) Symfony\Component\Process\Tests\ProcessTest::testWaitUntilSpecificOutput
Failed asserting that 4.3028318881989 is less than 2.
C:\projects\symfony\src\Symfony\Component\Process\Tests\ProcessTest.php:155
```